treeherder/model/migrations/0052_remove_unnecessary_job_indexes.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-01-25 18:01
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0051_add_job_lastmodified_index'),
+    ]
+
+    operations = [
```

Comment:
Checking prod I only see one index on `guid`, so perhaps Django was de-duping this already. Still keen to see the sqlmigrate output, in case Django drops and re-adds the existing index. (I've seen migrations that would drop and recreate an FK due to index changes, which would be painful, though I don't think that will affect this case).